components/Dialogs/src/ColorThemer/MDCAlertColorThemer.m

```diff
@@ -14,8 +14,8 @@
 
 #import "MDCAlertColorThemer.h"
 
-#import "../../../Buttons/src/ColorThemer/MaterialButtons+ColorThemer.h"
 #import "../MDCAlertController+ButtonForAction.h"
```

Comment:
Can we fix the relative path for "MDCAlertController+ButtonForAction"?

Reply:
Done